### ISSUE TITLE
Indicador with negative values

### ISSUE THREAD

**Issue by sheaur** [custom-cards/bar-card#181]
The indicator does not work correctly when the value is negative, example rssi

**Comment by sheaur** [comment]
![Uploading 20240516_030600.jpg…]()

**Comment by fwestenberg** [comment]
This would indeed be very nice if possible. I use it for my electricity power meter, when the solar panels produce power I'd like the bar to show something negative. Something like this would be nice:
```
target: '0'
min: '-5'
max: '5'
```

At this moment, it results in:

![image](https://github.com/user-attachments/assets/58b0555c-681c-4f0f-8d69-1337731a6d64)

**Comment by ChristopheVersieux** [comment]
Agreed, I am suprised I cannot find that anywhere on Home Assistant, this seems like a useful feature
I did like @fwestenberg did, but that's really not visual, colors should be centered on 0
